src/distrs/beta.jl

```diff
@@ -1,7 +1,6 @@
 # functions related to beta distributions
 
 import .RFunctions:
-    betalogpdf,
```

Comment:
I'm not sure what you mean when you say it won't overload, but if `Real` works and `Number` doesn't, I'd guess that it's because `Real` is more specific than `Number` so it's not overwriting an existing method if one for `Number` exists.